README.md

```diff
@@ -1,6 +1,26 @@
 # cluster-authentication-operator
+The authentication operator is an 
+[OpenShift ClusterOperator](https://github.com/openshift/enhancements/blob/master/enhancements/operator-dev-doc.md#what-is-an-openshift-clusteroperator).    
+It installs and maintains the Authentication [Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in a cluster and can be viewed with:     
```

Comment:
It is only partially maintaining the CR, but the main purpose of the CR is to be one of the configuration entrypoints for the operator.

`s/and can be viewed with/which can be viewed with` -> the original suggests the operator can be viewed by the command